<launch>
  <node pkg="tf" type="static_transform_publisher" name="map_to_robot_transform" args="1 1.5 0 0 0 0 /map_frame /robot_frame 100"/>
  <node pkg="tf" type="static_transform_publisher" name="map_to_goal_transform" args="1 2 3 0 0 0 /map_frame /goal_frame 100"/>
  <rosparam command="load" file="$(find 3d_navigation_ros_wrapper)/config/params.yaml" />
  <node pkg="3d_navigation_ros_wrapper" type="global_planner_node" name="global_planner" output="screen"/>
</launch> 
